### PR TITLE
SWG-275 - Moved installer code to shell script.

### PR DESCRIPTION
**Pull request by kathyoungaligent**

### CODE REVIEW

File: shell/clean_duplicates.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<?php
```
Comment (michaelwylde) [inline shell/clean_duplicates.php]: Does this need a modman entry to symlink to the correct location or is it expected to run from the vendor dir?

Comment (kathyoungaligent) [reply shell/clean_duplicates.php]: I'm an idiot.

Comment (kathyoungaligent) [reply shell/clean_duplicates.php]: Actually...modman already has `shell/* shell` - will that pick it up?

Comment (michaelwylde) [reply shell/clean_duplicates.php]: *shrug* when you remove the module and composer install does it setup the symlink correctly?

**Pull request merged**